common/components/filter/_filter.scss

```diff
@@ -0,0 +1,67 @@
+.app-filter {
+  margin: 0 0 govuk-spacing(3);
+}
+
+.app-filter__list {
+  @include govuk-clearfix;
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.app-filter__list-item {
+  flex-grow: 1;
+  min-height: 70px;
+  display: flex;
+}
+
+.app-filter__list-item--active {
+  background: govuk-colour('white');
+  border: 2px solid govuk-colour('black');
+  padding: govuk-spacing(4);
+  padding-right: govuk-spacing(6);
+  @include govuk-font($size: 19, $weight: normal, $line-height: 1.25);
+}
+
+.app-filter__link {
+  // cannot remove text-decoration from parts of the link, so we remove it everywhere and re-add it when necessary
+  text-decoration: none;
+  background: govuk-colour('blue');
+  border: 2px solid govuk-colour('blue');
+  color: govuk-colour('white');
+  display: block;
+  padding: govuk-spacing(4);
+  padding-right: govuk-spacing(6);
+  flex-grow: 1;
+
+  @include govuk-font($size: 19, $weight: normal, $line-height: 1.25);
+
+  &:link,
+  &:active,
+  &:visited {
+    color: govuk-colour('white');
+  }
+
+  &:hover {
+    color: #d5e8f3;
```

Comment:
This doesn't look quite right I don't think:

![image](https://user-images.githubusercontent.com/3327997/76512309-5388fc80-644c-11ea-8e58-2e0b66b3e9ce.png)

He's the equivalent on Notify which feels a little closer to what we want:
![image](https://user-images.githubusercontent.com/3327997/76512355-67346300-644c-11ea-88fa-176040f2cdea.png)

Reply:
Feels like that hex code may also already be a variable with `govuk-colour`?

Reply:
It doesn't -- but has a close-ish match which I am using now.